x-pack/plugins/security_solution/public/management/links.ts

```diff
@@ -79,33 +103,80 @@ export const links: LinkItem = {
     {
       id: SecurityPageName.policies,
       title: POLICIES,
+      description: i18n.translate('xpack.securitySolution.appLinks.policiesDescription', {
+        defaultMessage:
+          'Use policies to customize endpoint and cloud workload protections and other configurations.',
+      }),
+      landingIcon: IconEndpointPolicies,
       path: POLICIES_PATH,
       skipUrlState: true,
       experimentalKey: 'policyListEnabled',
     },
     {
       id: SecurityPageName.trustedApps,
       title: TRUSTED_APPLICATIONS,
+      description: i18n.translate(
+        'xpack.securitySolution.appLinks.trustedApplicationsDescription',
+        {
+          defaultMessage:
+            'Improve performance or alleviate conflicts with other applications running on your hosts.',
+        }
+      ),
+      landingIcon: IconTrustedApplications,
       path: TRUSTED_APPS_PATH,
       skipUrlState: true,
     },
     {
       id: SecurityPageName.eventFilters,
       title: EVENT_FILTERS,
+      description: i18n.translate('xpack.securitySolution.appLinks.eventFiltersDescription', {
+        defaultMessage: 'Exclude unwanted applications from running on your hosts.',
```

Comment:
This message should be for `blocklists` just as @dasansol92 pointed out. The mock seems to have swapped the messages for event filters and blocklists. Here's a screenshot of the existing event filters list page. 

![Screenshot 2022-05-17 at 10 22 26](https://user-images.githubusercontent.com/1849116/168765459-49006edc-6185-40d9-a9e5-221d45fbee72.png)